.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -9,6 +9,7 @@ findbugsFile = env.FINDBUGS_FILE?.trim() ?: null
 pr_type = env.PR_TYPE?.trim() ?: null
 build_jdk_tool = env.BUILD_JDK_TOOL?.trim() ?: 'kie-jdk11'
 build_maven_tool = env.BUILD_MAVEN_TOOL?.trim() ?: 'kie-maven-3.8.1'
+buildChainVersion = '2.3.21'
```

Comment:
Is it possible to use a single property/variable so that we don't need to manually keep in sync this value and the [GitHub Action one](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/.ci/actions/build-chain/action.yml#L33)?
If not I think we should add a comment here (and same in the GHA file) to mention the other property to be updated

Reply:
I was thinking to use a groovy script to get version from file like `.ci/actions/build-chain/action.yml`
let me push a generic method to https://github.com/kiegroup/jenkins-pipeline-shared-libraries we will reuse from kogito-pipelines

Reply:
proposed by https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/168

Reply:
change tested and finish, please review @danielezonca